laplace/subnetmask.py

```diff
@@ -0,0 +1,214 @@
+import torch
+from torch.nn.utils import parameters_to_vector
+
+from laplace.feature_extractor import FeatureExtractor
+
+__all__ = ['SubnetMask', 'RandomSubnetMask', 'LargestMagnitudeSubnetMask', 'LastLayerSubnetMask', 'LargestVarianceDiagLaplaceSubnetMask']
```

Comment:
Formatting looks weird on github. Looks like larger indent than in the other files.

Reply:
Yes this might be because I was using tabs (= 4 spaces) instead of spaces, which the remaining code uses? I'll change to spaces.

Reply:
Done